### PR TITLE
fix(web,renderer-app): room type tag error of the MainRoomList components

### PR DESCRIPTION
**Pull request by Cheerego7**

### CODE REVIEW

File: web/flat-web/src/pages/HomePage/MainRoomListPanel/MainRoomList.tsx [hunk 1]
```diff
@@ -120,7 +120,10 @@ export const MainRoomList = observer<MainRoomListProps>(function MainRoomList({
                         let primaryAction: RoomListItemButton<"replay" | "join" | "begin">;
                         switch (roomStatus) {
                             case RoomStatus.Idle: {
-                                primaryAction = { key: "begin", text: t("begin") };
+                                const isCreator = room.ownerUUID === globalStore.userUUID;
+                                isCreator
+                                    ? (primaryAction = { key: "begin", text: t("begin") })
+                                    : (primaryAction = { key: "join", text: t("join") });
```
Comment (BlackHole1) [inline web/flat-web/src/pages/HomePage/MainRoomListPanel/MainRoomList.tsx]:
```suggestion
                                primaryAction = isCreator ? {
                                    key: "begin",
                                    text: t("begin"),
                                } : {
                                    key: "join",
                                    text: t("join"),
                                };
```

File: desktop/renderer-app/src/pages/HomePage/MainRoomListPanel/MainRoomList.tsx [hunk 2]
```diff
@@ -121,7 +121,10 @@ export const MainRoomList = observer<MainRoomListProps>(function MainRoomList({
                         let primaryAction: RoomListItemButton<"replay" | "join" | "begin">;
                         switch (roomStatus) {
                             case RoomStatus.Idle: {
-                                primaryAction = { key: "begin", text: t("begin") };
+                                const isCreator = room.ownerUUID === globalStore.userUUID;
+                                isCreator
+                                    ? (primaryAction = { key: "begin", text: t("begin") })
+                                    : (primaryAction = { key: "join", text: t("join") });
```
Comment (BlackHole1) [inline desktop/renderer-app/src/pages/HomePage/MainRoomListPanel/MainRoomList.tsx]:
```suggestion
                                primaryAction = isCreator ? {
                                    key: "begin",
                                    text: t("begin"),
                                } : {
                                    key: "join",
                                    text: t("join"),
                                };
```

**Review by BlackHole1 — CHANGES_REQUESTED** [review]
There is a small flaw

File: desktop/renderer-app/src/pages/HomePage/MainRoomListPanel/MainRoomList.tsx [hunk 3]
```diff
@@ -121,7 +121,10 @@ export const MainRoomList = observer<MainRoomListProps>(function MainRoomList({
                         let primaryAction: RoomListItemButton<"replay" | "join" | "begin">;
                         switch (roomStatus) {
                             case RoomStatus.Idle: {
-                                primaryAction = { key: "begin", text: t("begin") };
+                                const isCreator = room.ownerUUID === globalStore.userUUID;
+                                primaryAction = isCreator
+                                    ? { key: "begin", text: t("begin") }
+                                    : (primaryAction = { key: "join", text: t("join") });
```
Comment (BlackHole1) [inline desktop/renderer-app/src/pages/HomePage/MainRoomListPanel/MainRoomList.tsx]: this is bug

**Review by BlackHole1 — CHANGES_REQUESTED** [review]

File: desktop/renderer-app/src/pages/HomePage/MainRoomListPanel/MainRoomList.tsx (hunk 3 above)
Comment (Cheerego7) [reply desktop/renderer-app/src/pages/HomePage/MainRoomListPanel/MainRoomList.tsx]: I'm drunk

**Review by BlackHole1 — APPROVED** [review]
LGTM

**Pull request merged**